### PR TITLE
Multitrait LT-FH++

### PR DESCRIPTION
**Pull request by JetteS**

### CODE REVIEW

File: R/Helper_functions.R [hunk 1]
```diff
@@ -294,11 +292,11 @@ convert_age_to_thresh = function(age, dist = "logistic", pop_prev = .1, mid_poin
...
   
   # Checking that age is valid
-  if(!is.numeric(age)) stop("The age must be numeric!")
-  if(any(age<0)) stop("The age must be non-negative!")
+  if(!is.numeric(age)&& !is.integer(age)) stop("The age must be numeric!")
+  if(age<0) stop("The age must be non-negative!")
   
   # Checking that dist is either logistic or normal.
-  if(is.character(dist)){
+  if(class(dist) == "character"){
```
Comment (EmilMiP) [inline R/Helper_functions.R]: change back

File: R/Helper_functions.R [hunk 2]
```diff
@@ -485,10 +480,10 @@ convert_liability_to_aoo = function(liability, dist = "logistic", pop_prev = .1,
                                     min_aoo = 10, max_aoo = 90, lower = stats::qnorm(0.05, lower.tail = F), upper = Inf ) {
   
   # Checking that liability is valid
-  if(!is.numeric(liability)) stop("The liability must be numeric!")
+  if(!is.numeric(liability) && !is.integer(liability)) stop("The liability must be numeric!")
   
   # Checking that dist is either logistic or normal.
-  if(is.character(dist)){
+  if(class(dist) == "character"){
```
Comment (EmilMiP) [inline R/Helper_functions.R]: change back

File: R/est_gen_liab_noFH_corr_trait.R [hunk 3]
```diff
@@ -123,9 +126,9 @@ estimate_liability_prevalence = function(status, sq.herits, genetic_corrmat, ful
 
   
   # Now we can extract the number of phenotypes
-  n_pheno <- nrow(sq.herits)
+  n_pheno <- nrow(h2_vec)
```
Comment (EmilMiP) [inline R/est_gen_liab_noFH_corr_trait.R]: should be length ?

File: R/Helper_functions.R (hunk 1 above)
Comment (JetteS) [reply R/Helper_functions.R]: Done

File: R/Helper_functions.R (hunk 2 above)
Comment (JetteS) [reply R/Helper_functions.R]: Done

File: R/est_gen_liab_noFH_corr_trait.R (hunk 3 above)
Comment (JetteS) [reply R/est_gen_liab_noFH_corr_trait.R]: Done

**Review by EmilMiP — APPROVED** [review]
looks good

**Pull request merged**